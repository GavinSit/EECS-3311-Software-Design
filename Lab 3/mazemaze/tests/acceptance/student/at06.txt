--ask for helps solving
new_game(easy)
solve
move(E)
move(E)
move(E)
move(E)
move(E)
move(E)
move(S)
move(S)
move(S)
move(S)
move(W)
move(S)
move(S)
move(E)

