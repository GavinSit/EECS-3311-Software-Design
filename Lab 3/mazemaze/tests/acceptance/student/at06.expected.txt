  State: 0 -> ok
->new_game(easy)
  State: 1 -> ok
  +-*-+---+---+---+---+---+---+
  | X                         |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 2 -> ok
  +-*-+---+---+---+---+---+---+
  | * * X                     |
  +   +   +   +---+---+   +   +
  |       |   |       |       |
  +   +---+   +---+   +---+   +
  |   |                   |   |
  +   +   +   +   +---+   +   +
  |   |       |   |       |   |
  +   +   +---+   +   +   +   +
  |       |   |   |           |
  +---+   +   +   +---+   +---+
  |   |   |       |       |   |
  +   +   +   +   +---+   +   +
  |       |                   |
  +---+---+---+---+---+---+- -+

  Game Number: 1
  Score: 0 / 1

->solve
  State: 3 -> ok
  +-*-+---+---+---+---+---+---+
  | * * X   o                 |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 4 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * X                 |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 5 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * X             |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 6 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * X         |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 7 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * X     |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 8 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * X |
  +   +   +   +---+---+   +   +
  |       | o |       |       |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 9 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     X |
  +   +---+   +---+   +---+   +
  |   |     o   o   o   o |   |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 10 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | X |
  +   +   +   +   +---+   +   +
  |   |       |   |     o |   |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 11 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | X |
  +   +   +---+   +   +   +   +
  |       |   |   |     o     |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 12 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | * |
  +   +   +---+   +   +   + * +
  |       |   |   |     o   X |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(W)
  State: 13 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | * |
  +   +   +---+   +   +   + * +
  |       |   |   |     X * * |
  +---+   +   +   +---+   +---+
  |   |   |       |     o |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 14 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | * |
  +   +   +---+   +   +   + * +
  |       |   |   |     * * * |
  +---+   +   +   +---+ * +---+
  |   |   |       |     X |   |
  +   +   +   +   +---+   +   +
  |       |             o   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(S)
  State: 15 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | * |
  +   +   +---+   +   +   + * +
  |       |   |   |     * * * |
  +---+   +   +   +---+ * +---+
  |   |   |       |     * |   |
  +   +   +   +   +---+ * +   +
  |       |             X   o |
  +---+---+---+---+---+---+- -+

  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

->move(E)
  State: 16 -> ok
  +-*-+---+---+---+---+---+---+
  | * * * * * * * * * * * * * |
  +   +   +   +---+---+   + * +
  |       | o |       |     * |
  +   +---+   +---+   +---+ * +
  |   |     o   o   o   o | * |
  +   +   +   +   +---+   + * +
  |   |       |   |     o | * |
  +   +   +---+   +   +   + * +
  |       |   |   |     * * * |
  +---+   +   +   +---+ * +---+
  |   |   |       |     * |   |
  +   +   +   +   +---+ * +   +
  |       |             * * * |
  +---+---+---+---+---+---+-X-+

  Congratulations! You escaped the maze!
  Since you used the solution, no points are awarded.
  Game Number: 1
  Score: 0 / 1

